.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I think here is "/node_modules"

Reply:
yes you're right
I will delete this file